.github/workflows/windows.yml

```diff
@@ -40,7 +40,7 @@ env:
   LAYOUT: "${{ contains(inputs.layout, '2020') && 'layout-2020' || 'layout-2013' }}"
   PACKAGE: "${{ contains(inputs.layout, '2020') && 'windows-msvc-layout2020' || 'windows-msvc' }}"
   CCACHE: "ccache"
-  CARGO_INCREMENTAL: 0
```

Comment:
Why are you removing `CARGO_INCREMENTAL` here?

Reply:
This was actually added while testing sccache, but as it is not working for windows builds it was removed except this line.

Reply:
But this change does not benefit performance, I had a commit to remove it before this somewhere.